src/components/Portfolio.js

```diff
@@ -1,7 +1,79 @@
 // Portfolio component - will be displayed when the Portfolio tab in main nav is clicked
+import Card from "./Card";
+import planet from "../images/planet.gif";
+import planner from "../images/planner-mockup.png";
+import weather from "../images/weather-dash.png";
+import pack from "../images/POT-login.png";
+import tech from "../images/tech-blog-dashboard.png";
+import note from "../images/note-taker-dashboard.png";
+//import cardImage from "../images/cardimage.jpg";
+
+const cardProps = [
```

Comment:
Might be worth renaming to e.g. projects, consider placing these in a 'constants' folder if they do not need to be declared in this exact file.